README.md

```diff
@@ -81,7 +81,7 @@ Once you have successfully fetched the players from the server, display them on
 
 **YOUR NOTES**
 ```
-
+I'm getting a 'GET http://127.0.0.1:5501/data.js net::ERR_ABORTED 404 (Not Found)' error but I was still getting everything to work
```

Comment:
To fix this, you'd need to open the [`index.html`](https://github.com/ihollander/js-mini-challenge-fetch/blob/c96ed21038e6be43b2be832ddad947dd95f5230f/index.html#L6) file and remove the script tag with the `data.js`